tests/Rules/UnusedPublicClassMethodRule/UnusedPublicClassMethodRuleTest.php

```diff
@@ -47,16 +48,23 @@ public static function provideDataTests(): Iterator
         yield [[__DIR__ . '/Fixture/Tests/SkipTestPublicMethod.php'], []];
         yield [[__DIR__ . '/Fixture/Tests/SkipTestCasePublicMethod.php'], []];
 
-        // report only the non-called static call
-        $errorMessage = sprintf(
+        $errorMessage1 = sprintf(
             UnusedPublicClassMethodRule::ERROR_MESSAGE,
             MethodForTests::class,
             'notCalledStaticCall'
         );
+        $errorMessage2 = sprintf(
+            UnusedPublicClassMethodRule::ERROR_MESSAGE,
+            MethodForTests::class,
+            'calledStaticCall'
+        );
         yield [[
             __DIR__ . '/Fixture/Tests/MethodForTests.php',
             __DIR__ . '/Fixture/Tests/SkipCalledInTests.php',
-        ], [[$errorMessage, 13, RuleTips::SOLUTION_MESSAGE]]];
```

Comment:
I think this test was wrong before and reporting 2 errors is correct, as the only calls are in Test-classes